lib/Controller/PersonController.php

```diff
@@ -161,8 +209,10 @@ private function getRedirectToFileUrl($fileId) {
 		$files      = $baseFolder->getById($fileId);
 		$file       = current($files);
 
-		if(!($file instanceof File))
+		if(!($file instanceof File)) {
+			$this->config->setUserValue($this->userId, 'facerecognition', StaleImagesRemovalTask::STALE_IMAGES_REMOVAL_NEEDED_KEY, 'true');
```

Comment:
maybe write some comment why is this key needed (and what `$file` can be if not `File`). I think it might be beneficial to "future us":)